rest/http_rest_service.cc

```diff
@@ -89,8 +89,7 @@ HttpRestConnection::
 sendErrorResponse(int responseCode, const Json::Value & error)
 {
     using namespace std;
-    cerr << "sent error response " << responseCode << " " << error
-         << endl;
+    cerr << "sent error response " << responseCode << " " << error << endl;
```

Comment:
I targeted messages that we shown at boot time.  This one can be done next time I touch that file!  :-)